ql/src/java/org/apache/hadoop/hive/ql/exec/ReplCopyTask.java

```diff
@@ -166,6 +169,12 @@ public int execute() {
     }
   }
 
+  private void setupWork() throws HiveException {
```

Comment:
All of these setupWork() implementations are basically the same, can they be moved to base class Task?

Reply:
The implementation of setupWork for each task seem to be updating different fields of the task class.